api/src/pcapi/core/offers/api.py

```diff
@@ -1092,3 +1092,26 @@ def create_educational_shadow_stock_and_set_offer_showcase(
     search.async_index_offer_ids([offer.id])
 
     return stock
+
+
+def transform_shadow_stock_into_educational_stock(
+    stock_id: str, stock_data: EducationalStockCreationBodyModel, user: User
+) -> Stock:
+    offer = offers_repository.get_educational_offer_by_id((stock_data.offer_id))
+
+    if offer.extraData["isShowcase"] != True:
```

Comment:
`is not True` est plus "Pythonesque" :)

Reply:
Est-ce qu'on pourrait avoir des cas où la clé est inexistante (après rattrapage des données)? Je ne pense pas puisqu'on la crée à la création du shadow stock, systématiquement (car si c'est le cas ça crash)
Du coup peut-être faire un `offer.extraData.get("isShowcase")` histoire d'être sereins?